AssemblyAutoBoard/AssemblyAutoBoard.ino

```diff
@@ -195,40 +193,39 @@ void loop() {
   //Main Timer to keep track of entire machine!
   unsigned long currentTime = millis();
   //Call LCD Clear function to clear 4th line of LCD
-  lcdClear();
+  lcdControl();
```

Comment:
Went to lcdControl to provide support for RunTime function while inside the mpsInput function.